src/main/java/com/agilethought/internship/sso/exception/HttpMessageNotReadableException.java

```diff
@@ -1,27 +0,0 @@
-/*package com.agilethought.internship.sso.exception;
```

Comment:
Don't remove it

Reply:
I delete this file (`exception/HttpMessageNotReadableException.java `) because it was all commented, and the `errorhandling/` files I decided to made an enum